src/backend/api.js

```diff
@@ -87,3 +87,21 @@ export const wikiDiscovery = async ({ sort, direction, active, currentPage, resu
     }
   })).data
 }
+
+export const importEntities = async ({
+  wikiId,
+  entityIds = ['P22', 'P25', 'P40', 'P18', 'P31', 'P279', 'Q1', 'Q2', 'Q64', 'Q42', 'Q3107329'],
```

Comment:
Not entirely sure if this function's default args are the right place to put our preset's configuration for now?